.gitignore

```diff
@@ -3,6 +3,7 @@
 # compiled output
 /dist
 /tmp
+/build
```

Comment:
I commit `build` folder by intention - this `pwa-workshop` is for "generic" JavaScript workshops (unlike `pwa-workshop-angular`), without need to have `angular-cli` installed.